helm-framework/helm/provider.go

```diff
@@ -641,3 +642,29 @@ func OCIRegistryPerformLogin(ctx context.Context, registryClient *registry.Clien
 	tflog.Info(ctx, fmt.Sprintf("Logged into OCI registry %q", u.Host))
 	return nil
 }
+
+// GetHelmConfiguration retrieves the Helm configuration for a given namespace
+func (m *Meta) GetHelmConfiguration(namespace string) (*action.Configuration, error) {
+	m.Lock()
+	defer m.Unlock()
+	tflog.Info(context.Background(), "[INFO] GetHelmConfiguration start")
```

Comment:
I see both `log` and `tflog` are used. I think we want to make this consistent and use `tflog` everywhere.